ament_cmake_gmock/ament_cmake_gmock-extras.cmake

```diff
@@ -21,11 +21,17 @@ macro(_ament_cmake_gmock_find_gmock)
 
     find_package(ament_cmake_test QUIET REQUIRED)
 
+    set(_prefix "${ament_cmake_gmock_DIR}/../../..")
```

Comment:
Since the `_prefix` variable is only used in on branch later it should also only be defined there.